.github/workflows/ci.yaml

```diff
@@ -43,20 +43,13 @@ on:
         default: Github actions job result
         type: string
     secrets:
-      mail_username:
```

Comment:
Why they are removed? You still use them

Reply:
Fixed